src/time.cpp

```diff
@@ -17,118 +17,167 @@
 
 #include "meep.hpp"
 
+#include <algorithm>
+#include <iterator>
+#include <map>
+
 using namespace std;
 
 namespace meep {
 
+namespace {
+
+constexpr size_t kMeepTimingStackSize = 10;
+
+const std::map<time_sink, const char *> kDescriptionByTimeSink{
```

Comment:
We don't actually use the Hungarian naming style anywhere else in Meep, so the `k` here (which I guess is part of the Google Coding Standards) is a bit incongruous.  `DescriptionByTimeSink` should be fine.

Reply:
Updated.